web/elm/src/SideBar/SideBar.elm

```diff
@@ -298,6 +298,25 @@ tooltip { hovered } =
                 , containerAttrs = Just Styles.tooltipBody
                 }
 
+        HoverState.Tooltip SideBarIcon _ ->
+            let
+                text =
+                    if sideBarState.isOpen then
```

Comment:
Minor thing, but when there are no pipelines (or e.g. you haven't logged in yet and there are no exposed pipelines), you can't actually click on the sidebar icon. Perhaps we shouldn't brighten the icon when you hover over but the icon is not clickable, and perhaps we should give a helpful tooltip (e.g. the sidebar cannot be expanded when there are no pipelines)

<img width="1009" alt="Screen Shot 2021-02-08 at 11 24 58 AM" src="https://user-images.githubusercontent.com/26583442/107248795-a96d7800-6a00-11eb-921b-0a6b4b72d333.png">
(can hover over and it suggests that you can expand the sidebar, but you cannot)

Or even weirder
<img width="574" alt="Screen Shot 2021-02-08 at 11 29 01 AM" src="https://user-images.githubusercontent.com/26583442/107249575-7d9ec200-6a01-11eb-8eb6-838f1a41c6b5.png">
(sidebar was open before logging out/the session expiring, so it displays the hide sidebar icon, but since there are no pipelines the sidebar is hidden)